ml-agents/mlagents/trainers/trainer.py

```diff
@@ -78,41 +88,60 @@ def dict_to_str(self, param_dict, num_tabs):
                 ]
             )
 
+    def __str__(self):
```

Comment:
Can some non-SAC changes like this be done in a preliminary PR (just to cut down on the line count in this one)?

Reply:
Yep - will be creating a trainer/model refactor PR